akka-docs/src/main/categories/fan-out-operators.md

```diff
@@ -1,2 +1,4 @@
 These have one input and multiple outputs. They might route the elements between different outputs, or emit elements on
-multiple outputs at the same time.
\ No newline at end of file
+multiple outputs at the same time.
+
+There is currently no 'fluent' API available for the fan-out operators, to use them you will have to use the @ref[Graph DSL](../stream-graphs.md#constructing-graphs).
```

Comment:
Actually, at few of them, or special cases of them, do have 'fluent' API: `alsoTo`, `divertTo` and `wireTap`.

Not sure if that is worth mentioning.

Reply:
These are currently filed under 'Simple operators'. I agree 'Fan-out operators' is more specific and thus better, I'll move them